modules/applications-create-using-cli-modify.adoc

```diff
@@ -199,3 +199,17 @@ To search for images, templates, and other inputs for the `oc new-app` command,
 ----
 $ oc new-app --search php
 ----
+
+== Setting the import mode 
+
+To set the import mode when using `oc new-app`, add the `--import-mode` flag. This flag can be appended with `Legacy` or `PreserveOriginal`, which provides users the option to trigger builds using a single sub-manifest, or all manifests, respectively. 
```

Comment:
@stevsmit Hi, we said here to trigger build, but the example just to create imagestream and deployment, do we need update this description or the example cmds?

The example cmd only create imagestream and deployment
```oc new-app --image=registry.redhat.io/ubi8/httpd-24:latest  --import-mode=PreserveOriginal --name=test
error: unknown flag: --import-mode
See 'oc new-app --help' for usage.
xiuwang@Xiuwangs-MacBook-Pro 414 % ./oc new-app --image=registry.redhat.io/ubi8/httpd-24:latest  --import-mode=PreserveOriginal --name=test
--> Found container image 13086f1 (5 seconds old) from registry.redhat.io for "registry.redhat.io/ubi8/httpd-24:latest"

    * An image stream tag will be created as "test:latest" that will track this image

--> Creating resources ...
    imagestream.image.openshift.io "test" created
    deployment.apps "test" created
--> Success
    Run 'oc status' to view your app.
```